pyogrio/geopandas.py

```diff
@@ -152,6 +154,12 @@ def write_dataframe(df, path, layer=None, driver=None, encoding=None, **kwargs):
     encoding : str, optional (default: None)
         If present, will be used as the encoding for writing string values to
         the file.
+    geometry_type : string, optional (default: None)
+        The geometry type for the dataset layer that will be written.
+        By default will be inferred from the data, but this parameter allows
+        to override this and specify the geometry type manually. Possible
+        values: 'Unknown', 'Point', 'LineString', 'Polygon', 'MultiPoint',
+        'MultiLineString', 'MultiPolygon', 'GeometryCollection'.
```

Comment:
Do we want to add a note stating that using an incorrect geometry type may prevent writing the data to certain drivers or cause performance issues when used within other applications?